fitbenchmarking/parsing/fitbenchmark_parser.py

```diff
@@ -165,7 +164,8 @@ def _set_data_points(self, data_points: list, fit_ranges: list) -> None:
             self.fitting_problem.start_x = fit_ranges[0]['x'][0]
             self.fitting_problem.end_x = fit_ranges[0]['x'][1]
 
-    def _set_additional_info(self) -> None:
+    @staticmethod
+    def _set_additional_info() -> None:
```

Comment:
I think this one would be better as a pylint ignore.
The intention is to have an interface for subclasses to use to make extra notes on self.fitting_problem. I think it's just mantid that uses it. Maybe it needs a rethink though.